infra-templates/k8s/18/rancher-compose.yml

```diff
@@ -31,6 +31,21 @@
     options:
      - rancher
      - aws
+  - variable: DISABLE_ADDONS
+    label: Disable rancher addons
+    description: Warning, Setting this to true will disable dashboard, helm and other k8s services
```

Comment:
we should mention that it will disable skydns as well. And we should mention that the main purpose of disabling it will be user wanting to implement his own version of the services